cli/azd/internal/repository/app_init.go

```diff
@@ -768,3 +789,35 @@ func ServiceFromDetect(
 
 	return svc, nil
 }
+
+func processSpringCloudAzureDepByPrompt(console input.Console, ctx context.Context, project *appdetect.Project) error {
+	continueOption, err := console.Select(ctx, input.ConsoleOptions{
+		Message: "Detected Kafka dependency but no spring-cloud-azure-starter found. Select an option",
+		Options: []string{
+			"Exit then I will manually add this dependency",
+			"Continue and do not add this dependency",
+			"Continue and do not provision Azure Event Hubs for Kafka",
```

Comment:
1. `Continue and do not add this dependency` -> `Continue without this dependency, and provision Azure Event Hubs for Kafka`
2. `Continue and do not provision Azure Event Hubs for Kafka` -> `Continue without this dependency, and not provision Azure Event Hubs for Kafka`